### ISSUE TITLE
How to handle buggy Augustus gffs

### ISSUE THREAD

**Issue by thackl** [thackl/gggenomes#65]
Came across this when working on #64 with Augustus example data. Related to https://github.com/Gaius-Augustus/Augustus/issues/74. 

Augustus does not set CDS and exon IDs correctly - one ID per line/range instead of one ID for all lines belonging to the same connected CDS/exons. This messes with the plots, because I assume that every CDS with a unique ID is a complete CDS and thus should have its own mRNA... 

Not sure how and to what extent to address this within gggenomes. Some fixes are described here https://github.com/Gaius-Augustus/Augustus/issues/74

``` {r}
f0 <- read_gff3("aten_0.1.m1.802.gff3")
gggenomes(f0) + geom_gene(position="pile")
```

![augustus-cds-issue](https://user-images.githubusercontent.com/9363087/128674251-2eab67e4-bd49-4cb0-9f59-886356b358fa.png)

**Comment by thackl** [comment]
To be more precise - exons **are** single-range features. Each exon should have its own ID. However, CDS that are part of the same proteins should be linked by having the same ID.

**Comment by thackl** [comment]
Fixed by 250563b207abd621a48d8683d25dce17ea4f0a09 which adds `read_gff3(..., fix_augustus_cds=TRUE)`
